tests/backends/pyq/test_time_dependent_generator.py

```diff
@@ -25,24 +25,79 @@ def test_time_dependent_generator(
 ) -> None:
     n_steps = 500
 
+    psi_0 = qutip.basis(4, 0)
+    psi0_torch = (torch.tensor(psi_0.full()).to(torch.complex128)).reshape(1, -1)
```

Comment:
Why do you have to use a qutip object here ? Ah I see. I'd suggest to add them as fixtures.

Reply:
Eventually I could remove the initial state `psi0_torch` as it corresponds to the state=None.

Reply:
Why adding qutip object as fixture?

Reply:
Ok eventually skipping this as tests changed